src/main/java/org/broadinstitute/hellbender/tools/exome/TargetCoverageStatsReader.java

```diff
@@ -61,7 +61,7 @@ public TargetCoverageStatsReader(final File outputFile) throws IOException {
                             dataLine.getInt(endColumnIndex)));
 
         } else {
-            final int nameColumnIndex = columns.indexOf(TargetTableColumns.NAME.name());
```

Comment:
This class is not related to the deceased `TargetCoverage`.  It's part of the germline pipeline for filtering targets, which itself will probably be on the chopping block soon, but not yet.